packages/app-mobile/components/NoteEditor/CodeMirror/CodeMirror.ts

```diff
@@ -183,7 +71,12 @@ export function initCodeMirror(parentElement: any, initialText: string, theme: a
 			// See https://github.com/codemirror/basic-setup/blob/main/src/codemirror.ts
 			// for a sample configuration.
 			extensions: [
-				markdown(),
+				markdown({
+					extensions: [
+						MarkdownMathExtension,
+						GitHubFlavoredMarkdownExtension,
```

Comment:
The `GitHubFlavoredMarkdownExtension` is added here to prevent a regression.

Note that connecting the `MarkdownMathExtension` should produce no visible changes — we haven't defined syntax highlighting rules for math regions yet.

The `MarkdownMathExtension`, however, is being used as a way of testing the `jest` setup (and decreasing the amount of code to review in future PRs). As such, this connection might be outside of the scope of this pull request.